cmd/gardener-admission-controller/app/gardener_admission_controller.go

```diff
@@ -115,6 +116,8 @@ func (o *options) validate() error {
 }
 
 func (o *options) run(ctx context.Context) error {
+	logf.SetLogger(logger.MustNewZapLogger(o.config.LogLevel, o.config.LogFormat))
```

Comment:
Do not use `Must`, do it like here: https://github.com/gardener/gardener/blob/1da39002ccd90e8b3dc08763dcf41708e48d907f/cmd/gardener-controller-manager/app/app.go#L67-L70